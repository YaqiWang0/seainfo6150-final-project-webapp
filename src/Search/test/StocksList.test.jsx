import React from "react";
import data from "./data";
import StocksList from "../Stocks/StockLists";

describe("StocksList test", () => {

    it('render successfully', function () {
        const {container} = render(
            <StocksList stocks = {data} test = {true}/>
        );
        expect(container).toMatchSnapshot();
    })
})