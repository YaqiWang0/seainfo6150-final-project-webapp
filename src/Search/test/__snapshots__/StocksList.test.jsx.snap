// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StocksList test render successfully 1`] = `
<div>
  <div
    class="container"
  >
    <div>
      <div
        class="container1"
      >
        <div
          class="name"
        >
          <span
            class="stock_name"
          >
            TSLA
          </span>
          <span
            class="company_name"
          >
            Tesla, Inc.
          </span>
        </div>
      </div>
    </div>
    <div>
      <div
        class="container1"
      >
        <div
          class="name"
        >
          <span
            class="stock_name"
          >
            MTSL
          </span>
          <span
            class="company_name"
          >
            Mer Telemanagement Solutions Ltd.
          </span>
        </div>
      </div>
    </div>
    <div>
      <div
        class="container1"
      >
        <div
          class="name"
        >
          <span
            class="stock_name"
          >
            FTSL
          </span>
          <span
            class="company_name"
          >
            First Trust Senior Loan Fund
          </span>
        </div>
      </div>
    </div>
  </div>
</div>
`;
